Packs/SentinelOne/Integrations/SentinelOne-V2/SentinelOne-V2.py

```diff
@@ -3434,8 +3433,8 @@ def main():
     fetch_type = params.get('fetch_type', 'Threats')
     first_fetch_time = params.get('fetch_time', '3 days')
     fetch_severity = params.get('fetch_severity', [])
-    fetch_incidentStatus = params.get('fetch_incidentStatus', [])
-    fetch_threat_incident_statuses = params.get('fetch_threat_incident_statuses', [])
+    fetch_incidentStatus = params.get('fetch_incidentStatus', ["UNRESOLVED"])
+    fetch_threat_incident_statuses = params.get('fetch_threat_incident_statuses', ["UNRESOLVED"])
```

Comment:
Is `fetch_incidentStatus` (if there is a value) not a string?
Why do you give a list by default?
I think you can use the [argToList](https://github.com/demisto/content/blob/master/Packs/Base/Scripts/CommonServerPython/CommonServerPython.py#L1824) function in such a case

Reply:
If there is a value in these fields of config`fetch_incidentStatus` and `fetch_threat_incident_statuses` it should be a list. Provided default value if we didn't found the fields in the params, in order to avoid breakage of the code flow.
Example: If we select the IN_PROGRESS, UNRESOLVED and RESOLVED while configuring the value of the above fields  will be ['IN_PROGRESS', 'RESOLVED', 'UNRESOLVED'].